game-core/src/main/java/games/strategy/triplea/ResourceLoader.java

```diff
@@ -169,43 +170,11 @@ static String normalizeMapName(final String zipName) {
     return sb.toString();
   }
 
-  private static List<String> getPaths(final String mapName) {
-    if (mapName == null) {
-      return new ArrayList<>();
-    }
-
-    final List<File> candidates = getCandidatePaths(mapName);
-
-    final Optional<File> match = candidates.stream().filter(File::exists).findFirst();
-    if (match.isEmpty()) {
-      // if we get no results, we will eventually prompt the user to download the map
-      return new ArrayList<>();
-    }
-
-    final List<String> paths = new ArrayList<>();
-    paths.add(match.get().getAbsolutePath());
-    // find dependencies
-    try (URLClassLoader url = new URLClassLoader(new URL[] {match.get().toURI().toURL()})) {
-      final URL dependenciesUrl = url.getResource("dependencies.txt");
```

Comment:
Wait what?
Does any map even use this?

Reply:
I've all maps cloned to "work/maps"

```
dan@dan-xps:~$ cd work/maps/
dan@dan-xps:~/work/maps$ find . -name "dependencies.txt"
dan@dan-xps:~/work/maps$ 
```

I was a bit surprised. I wonder if maybe back when we had 'variants', if it was used then. We've since merged all of the core map data into each variant to remove the variant maps and have them as stand-alone. If that is the case, it explains this could having been used but no longer.